src/local_scheduler/format/local_scheduler.fbs

```diff
@@ -17,9 +17,12 @@ enum MessageType:int {
   // Send a reply confirming the successful registration of a worker or driver.
   // This is sent from the local scheduler to a worker or driver.
   RegisterClientReply,
-  // Notify the local scheduler that this client is disconnecting gracefully.
+  // Notify the local scheduler that this client is disconnecting unexpectedly.
```

Comment:
Let's reword this comment a little bit. `Notify the local scheduler that the client has disconnected unexpectedly.`